docs/user-guide/configuration.md

```diff
@@ -205,25 +240,36 @@ index.md     | index.html           | /                        | /index.html
 api-guide.md | api-guide/index.html | /api-guide/              | /api-guide/index.html
 about.md     | about/index.html     | /about/                  | /about/index.html
 
-The default style of `use_directory_urls=true` creates more user friendly URLs, and is usually what you'll want to use.
+The default style of `use_directory_urls=true` creates more user friendly URLs,
+and is usually what you'll want to use.
 
-The alternate style can occasionally be useful if you want your documentation to remain properly linked when opening pages directly from the file system, because it create links that point directly to the target *file* rather than the target *directory*.
+The alternate style can occasionally be useful if you want your documentation to
+remain properly linked when opening pages directly from the file system, because
+it create links that point directly to the target *file* rather than the target
+*directory*.
 
 **default**: `true`
 
 ### strict
 
-Determines if a broken link to a page within the documentation is considered a warning or an error (link to a page not listed in the pages setting).  Set to true to halt processing when a broken link is found, false prints a warning.
+Determines if a broken link to a page within the documentation is considered a
+warning or an error (link to a page not listed in the pages setting). Set to
+true to halt processing when a broken link is found, false prints a warning.
 
 **default**: `false`
 
 ### dev_addr
 
-Determines the address used when running `mkdocs serve`.  Setting this allows you to use another port, or allows you to make the service accessible over your local network by using the `0.0.0.0` address.
+Determines the address used when running `mkdocs serve`. Setting this allows you
+to use another port, or allows you to make the service accessible over your
+local network by using the `0.0.0.0` address.
 
-As with all settings, you can set this from the command line, which can be useful, for example:
+As with all settings, you can set this from the command line, which can be
+useful, for example:
 
-    mkdocs serve --dev-addr=0.0.0.0:80  # Run on port 80, accessible over the local network.
+```bash
+mkdocs serve --dev-addr=0.0.0.0:80  # Run on port 80, on the local network.
```

Comment:
Unfortunately, the linter seems to enforce the line length on code blocks also. While generally not  a bad idea, sometimes, there may be a legitimate reason for it. In any event, I was able to shorted the comment without losing the meaning (I think), so no great loss here. If you disagree, please advise and we can improve it.